js/components/developer/networking-login-screen/networkingLogin.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+import { Content, Card, CardItem } from 'native-base';
+
+import EmailInput from '../../common/email-input';
+import PasswordInput from '../../common/password-input';
+import CardItemButton from '../../common/card-item-button/cardItemButton';
+
+import { signUp, signIn } from '../../../networking/auth';
+
+// Setting default values so they do not have to be entered every time
+const USER = 'noname@nomail.nope';
+const PASSWORD = 'password';
+
+class NetworkingLogin extends Component {
+  static propTypes = {
+    onLogin: React.PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: USER,
+      password: PASSWORD,
+      status: 'sign in or create account',
+    };
+  }
+
+  create() {
+    signUp(this.state.email, this.state.password,
+      () => alert('Account created'),
+      error => this.setState({ status: error.toString() }));
+  }
+
+  logIn() {
+    signIn(this.state.email, this.state.password,
+      (responseJson) => {
+        this.props.onLogin(responseJson.data.attributes['auth-token'],
```

Comment:
_FYI_ as mentioned before in order to use our API without going nuts (the API is _very_ verbose), we should add a layer of abstraction. You should also set the HTTP header `X-API-KEY-TRANSFORM` to `underscore`, since kebab-case attribute names is deprecated and only kept around for backwards compatibility with an old client that we still have (somewhat) running.

I'd be happy to point you to some options of what JSONAPI clients/parsers you can use (but trust me you need _something_, you can write it yourselves, but you need _something_/_anything_).